src/ui/linux/TogglDesktop/loginwidget.cpp

```diff
@@ -46,6 +46,20 @@ void LoginWidget::displayError(
     enableAllControls(true);
 }
 
+void LoginWidget::setGoogleSignup(bool visible) {
+    ui->email->setEnabled(!visible);
+    ui->password->setEnabled(!visible);
+    ui->password->clear();
+    if (visible) {
+        ui->email->setText("You're signing up with your Google Account");
```

Comment:
Look like this LOC add the Text into the Email textbox. Thus, I couldn't select it after failed authentication.

Reply:
This is now fixed by hiding the message when returning to the login screen.

Reply:
I changed the logic to ask for country and agree to the terms first and then enable opening the OAuth dialog. Resolving conversation.